src/server/src/app.ts

```diff
@@ -65,12 +61,13 @@ function connect(client: SocketIO.Socket) {
 }
 
 function initialize() {
+    let port = process.env.PORT || '5000';
```

Comment:
Op regel 30 wordt nog een poort 3000 gezet in het app object. Hier 5000...
Dat moet natuurlijk één default poort worden.